utilityScripts/createInstance.py

```diff
@@ -11,11 +11,11 @@
             region_name=Config.ec2_region
```

Comment:
There are no actual changes in this entire file, just remove it from your commit and revert this file back to the one currently in master.